docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -139,10 +144,13 @@ fields:
       - Education: ED-00-00-00-00
       - Prisoners' Rights: RI-10-00-00-00
       - Other: Other
+    none of the above: False
```

Comment:
Same here, none of the above should still be available. We don't have a good way to look up and learn more about [LIST](https://taxonomy.legal/) yet, and None of the Above is much quicker than forcing a weird Other selection.